apps/tng-dictionary-e2e/src/e2e/app.cy.ts

```diff
@@ -0,0 +1,13 @@
+import { getGreeting } from '../support/app.po';
+
+describe('tng-dictionary', () => {
+    beforeEach(() => cy.visit('/'));
+
+    it('should display welcome message', () => {
+        // Custom command example, see `../support/commands.ts` file
+        cy.login('my-email@something.com', 'myPassword');
```

Comment:
Is this for an admin panel or is this related to the API?

Reply:
This is just boilerplate that got moved under the migration. I'm guessing it was a `js` file before.